app/assets/stylesheets/homepage.scss

```diff
@@ -0,0 +1,47 @@
+$coral: #ec4339;
+$white: #ffffff;
+
+.title-box {
+  background-color: $coral;
+  font-family: 'Alegreya Sans SC', sans-serif;
+  color: $white;
+  padding: 10px;
+  font-weight: bold;
+  letter-spacing: 1.25px;
+  font-size: 60px;
+  font-size: 5vw;
+ }
+
+.vid-image {
+  width: -webkit-fit-content;
+  height: -webkit-fit-content;
+  width: -moz-fit-content;
+  height: -moz-fit-content;
+  position: relative;
+  background-position-x: 50%;
+}
+
+.vid-overlay {
+  bottom: 0;
+  color: $white;
+  display: none;
+  float: left;
+  left: 0;
```

Comment:
Properties should be ordered bottom, color, display, float, font-size, font-weight, left, padding-top, position, right, text-align, text-shadow, top